docs/docusaurus.config.js

```diff
@@ -115,7 +117,8 @@ module.exports = {
     },
     algolia: {
       // This API key is "search-only" and safe to be published
-      apiKey: "d58e0d68c875346d52645d68b13f3ac0",
+      apiKey: "011e01358301f5023b02da5db6af7f4d",
```

Comment:
For those looking at this in the future, yes the change in apikey was intentional. Still a search-only api key. Docusaurus required appId for this upgrade.